packages/backend/src/studio.spec.ts

```diff
@@ -55,6 +60,9 @@ afterEach(() => {
 });
 
 test('check activate ', async () => {
+  vi.spyOn(fs.promises, 'readFile').mockImplementation(() => {
+    return "<html></html>";
```

Comment:
```suggestion
    return '<html></html>';
```

(I suppose linter/formatter won't be happy)